bat-utils/lib/youtube.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const braveHapi = require('./extras-hapi')
+
+async function getYoutubeChannelId (youtubeUsername) {
+  const youtubeApiKey = process.env.YOUTUBE_API_KEY
+  const youtubeApiBaseUri = process.env.YOUTUBE_API_BASE_URI || 'https://www.googleapis.com'
+
+  let response = await braveHapi.wreck.get(`${youtubeApiBaseUri}/youtube/v3/channels?part=id&forUsername=${youtubeUsername}&key=${youtubeApiKey}`)
+  response = JSON.parse(response)
+
+  return response.items[0].id
+}
+
+module.exports = getYoutubeChannelId
```

Comment:
Should this be it's own file or just be a function extras-utils?

Reply:
i'm fine with either, even going down to one function per file

Reply:
radical